package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Project template for a FullStack project with NodeJS for backend and React for frontend",
   "scripts": {
-    "server-start-prod": "node server.js",
+    "server-start-prod": "NODE_ENV=production node server.js",
```

Comment:
Important to set the `NODE_ENV` environment variable to production when running in production mode